__tests__/app/api/chat/stream/route.test.ts

```
@@ -0,0 +1,251 @@
+      'data: chunk1\n\n',
+      'data: chunk2\n\n',
+    ]);
+    const mockResponse = {
+      status: 200,
+      statusText: 'OK',
+      headers: {},
+      ok: true,
+      cancel: jest.fn(),
+      [Symbol.asyncIterator]: mockStream[Symbol.asyncIterator],
+    };
+    mockedBackendRequest.mockResolvedValue(mockResponse as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)